helpers/response/create_response.py

```diff
@@ -1,52 +1,50 @@
 from datetime import datetime
-from graphql import GraphQLError
 from api.response.models import Response
 from api.room_resource.models import Resource
-from utilities.validations import (
-    validate_rating_field,
-    validate_missing_items_field)
 
 
-def create_response(question_type, **kwargs):
+def create_response(question_type, errors, responses, **kwargs):
     if question_type.lower() == 'rate':
-        if 'rate' not in kwargs:
-            raise GraphQLError("Provide a rating response")
-        else:
-            validate_rating_field(**kwargs)
+        if 'rate' in kwargs and kwargs['rate']:
+            rating = [1, 2, 3, 4, 5]
+            if kwargs['rate'] not in rating:
+                errors.append('Please rate between 1 and 5 for question {}'.format( # noqa
```

Comment:
Please have the `# noqa` outside the parentheses

Reply:
Thanks, it has been done.